pubspec.yaml

```diff
@@ -1,26 +1,28 @@
 name: transformer_utils
 version: 0.2.3
-description: Utilities relating to code generation, Dart analyzer, logging, etc. for
-    use in Pub transformers.
+description: Utilities relating to code generation, Dart analyzer, logging, etc. for use in Pub transformers.
 authors:
   - Workiva UI Platform Team <uip@workiva.com>
   - Workiva Client Platform Team <cp@workiva.com>
   - Greg Littlefield <greg.littlefield@workiva.com>
   - Evan Weible <evan.weible@workiva.com>
 homepage: https://github.com/Workiva/dart_transformer_utils
+
+environment:
+  sdk: '>=2.4.0 <3.0.0'
+
 dependencies:
-    analyzer: '>=0.35.0 <0.39.0'
-    build: ^1.0.0
-    dart2_constant: ^1.0.1
-    path: ^1.3.0
-    source_span: ^1.2.0
+  analyzer: ">=0.35.0 <0.39.0"
+  build: ^1.2.2
+  path: ^1.6.4
+  source_span: ^1.5.5
 
 dev_dependencies:
-    coverage: ">=0.10.0 <0.13.0"
-    dart_dev: ^2.0.0
-    dart_style: ^1.0.9
-    dependency_validator: ^1.1.0
-    test: ^1.5.3
-
-environment:
-    sdk: ">=2.0.0 <3.0.0"
+  build_runner: ^1.7.1
+  build_test: ^0.10.9
+  build_vm_compilers: ^1.0.3
+  build_web_compilers: ^2.5.1
```

Comment:
Do we need all of these `build_*` deps? It looks like we didn't have them before.

Reply:
I believe it's being added automatically with `d1down_update_pubspec`, but you right we can remove them